src/PointwiseFunctions/GeneralRelativity/WeylElectric.cpp

```diff
@@ -49,6 +49,40 @@ void weyl_electric(
     }
   }
 }
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+Scalar<DataType> weyl_electric_scalar(
+    const tnsr::ii<DataType, SpatialDim, Frame>& weyl_electric,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept {
+  Scalar<DataType> weyl_electric_scalar_part{};
+  weyl_electric_scalar<SpatialDim>(make_not_null(&weyl_electric_scalar_part),
```

Comment:
I would have thought this should be `gsl::make_not_null`, not `make_not_null`

Reply:
For some reason, changing this to `gsl::make_not_null` makes a bunch of compiler errors... it reccomends `gsl::not_null` which also yields similar errors. Do you have other suggestions regarding this? Thank you!

Reply:
You're right! I was too hasty in this comment. `make_not_null` is a spectre function defined in `Utilities/Gsl.hpp` that returns a pointer wrapped in `gsl::not_null`. Sorry for sending you on a wild goose chase! The way you originally had it is correct.